src/sentry/apidocs/parameters.py

```diff
@@ -547,12 +547,9 @@ class SessionsParams:
         required=True,
         type=str,
         many=True,
-        description="""The list of fields to query.\n\nThe available fields are\n  - `sum(session)`\n  - `count_unique("
-                    "user)`\n  - `avg`, `p50`, `p75`, `p90`, `p95`, `p99`, `max` applied to `session.duration`. For "
-                    "example, `p99(session.duration)`. Session duration is [no longer being recorded]("
-                    "https://github.com/getsentry/sentry/discussions/42716) as of on Jan 12, 2023. Returned data may "
-                    "be incomplete.\n  - `crash_rate`, `crash_free_rate` applied to `user` or `session`. For example, "
-                    "`crash_free_rate(user)`""",
+        description="""The list of fields to query.\n\nThe available fields are\n  - `sum(session)`\n  - `count_unique(user)`\n  - `avg`, `p50`, `p75`, `p90`, `p95`, `p99`, `max` applied to `session.duration`. For
```

Comment:
Here too, it might be a little easier to read to put the bullet points into separate lines like https://github.com/getsentry/sentry/blob/0a5744b69968ea082cdee668b906cd2a478bf560/src/sentry/apidocs/parameters.py#L49-L52